addons/web/static/src/core/py_js/py_date.js

```diff
@@ -362,6 +362,21 @@ export class PyDateTime {
     toJSON() {
         return this.strftime("%Y-%m-%d %H:%M:%S");
     }
+
+    to_utc() {
+        const d = new Date(this.year, this.month, this.day, this.hour, this.minute, this.second);
+        const offset = d.getTimezoneOffset();
+        // previous implementation did use timedelta
+        const s = tmxxx(
+            this.year,
+            this.month,
+            this.day,
+            this.hour,
+            this.minute,
+            this.second + 60 * offset
```

Comment:
Isn't it the opposite?

Reply:
I think I follow closely the former implementationf

Reply:
Maybe we should rely on timedelta as before but this would mean that we have to fix timedelta and relativedelta before to be able to fix the problems mentioned in the PR.